src/hooks/graphql/useLazyResultQuery.ts

```diff
@@ -24,7 +28,10 @@ const useLazyResultQuery = <TData = any, TVariables = OperationVariables>(
  *
  * see example here: /views/screens/variant/Entity/index.tsx
  */
-export const useLazyResultQueryOnLoadOnly = <TData = any, TVariables = OperationVariables>(
+export const useLazyResultQueryOnLoadOnly = <
+  TData = any,
+  TVariables extends OperationVariables = OperationVariables,
+>(
```

Comment:
To fix compilation error:
![F90FAFBE-A56F-4D6C-8AF6-1EE0C23B58E6_1_105_c](https://user-images.githubusercontent.com/116835792/222856065-1b8ade02-804c-46af-b080-5d27461a1077.jpeg)